### DISCUSSION TITLE
How can I use PSAT with Edge?

### DISCUSSION THREAD

**Discussion by andreylipattsev** [GoogleChromeLabs/ps-analysis-tool#547] [Support Forum] [answered]
Hi,
With Edge announcing third-party cookies phaseout / deprecation, can I use this tool in Edge too?

**Comment by amedina** [comment, marked as answer]
You can install PSAT on Edge directly from the Chrome WebStore. Currently PSAT's capabilities all work well on Edge, module some "localization" aspects where the keyword `Chrome` is used. 

This post explains the process: https://medium.com/@mariusbongarts/how-to-install-chrome-extensions-in-microsoft-edge-browsers-65914eb61d6
